pkg/core/resources/apis/mesh/mesh_helpers.go

```diff
@@ -36,3 +38,29 @@ func (m *MeshResource) GetTracingBackend(name string) *mesh_proto.TracingBackend
 	}
 	return backends[name]
 }
+
+// GetLoggingBackends will return logging backends as comma separated strings
+// if empty return default logging
+func (m *MeshResource) GetLoggingBackends() string {
+	var backends []string
+	if len(m.Spec.GetLogging().GetBackends()) != 0 {
+		for _, backend := range m.Spec.GetLogging().GetBackends() {
+			backends = append(backends, backend.GetName())
+		}
+		return strings.Join(backends, ", ")
+	}
+	return m.Spec.GetLogging().GetDefaultBackend()
```

Comment:
default backend is one of the existing backend. There is no situation that there are 0 backends and there is default backend. I think the logic should be just
```
var backends []string
for _, backend := range m.Spec.GetLogging().GetBackends() {
	backends = append(backends, backend.GetName())
}
return strings.Join(backends, ", ")
```
(it should return `""` when there are no backends)

Reply:
Same with `GetTracingBackends()`

Reply:
then, if it is empty means we need to show logging as "off" right as well for tracing ?

Reply:
Yes